### DISCUSSION TITLE
Show both the filter and the count visible in both view and edit mode

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#3758] [General]
Proposing we keep both the filter and the count visible in both view and edit mode. I wanted to use the filter feature today and for a moment thought my app was out of date because i didn't see the icon until i clicked into the BP. I think that'll make the filtering more discoverable and could improve some of the maxHits UX if we could allow editing if the conditional reduces the number of hits.

**Comment by jonbell-lot23** [comment]
Are we talking about the breakpoint feature?

<img width="517" alt="image" src="https://user-images.githubusercontent.com/9154902/158480770-18c57df1-7bce-40c2-aaa2-b84e06316516.png">

So we'd add a filter button next to the hits capsule?
